azure-iot-device/azure/iot/device/provisioning/models/registration_result.py

```diff
@@ -0,0 +1,109 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+
+class RegistrationResult(object):
+    """
+    The final result of a completed or failed registration attempt
+    :ivar:request_id: The request id to which the response is being obtained
+    :ivar:operation_id: The id of the operation as returned by the registration request.
+    :ivar status: The status of the registration process as returned by the Hub
+    Values can be "unassigned", "assigning", "assigned", "failed", "disabled"
+    :ivar registration_state : Details like device id, assigned hub , date times etc returned
+    from the provisioning service.
+    """
+
+    def __init__(self, rid, operation_id, status, registration_state=None):
+        """
+        :param rid: The request id to which the response is being obtained
```

Comment:
no param for registration_state

Reply:
you mean the one at line 25 ??

---
In reply to: [282977841](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r282977841) [](ancestors = 282977841)